examples/browsercontext_addinitscript.js

```diff
@@ -0,0 +1,32 @@
+import { check } from 'k6';
+import { chromium } from 'k6/x/browser';
+
+export const options = {
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function() {
+  const browser = chromium.launch({
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
+  await context.addInitScript(
+    `(function () {
+        document.open();
+        document.write("k6Test");
```

Comment:
I think we're going to have to restrict where we read/write files from/to. Even though the browser might be dockerised, it's worth mitigating issues by narrowing the scope of where customer's can read/write.